frontend/static/languages/irish_1k.json

```
@@ -0,0 +1,1023 @@
+    "chuireann",
+    "foraoisí",
+    "feachtas",
+    "sprioc",
+    "méid",
+    "gné",
+    "aithnítear",
+    "ghaeltachta",
+    "ceantair",
+    "tacaíocht",
+    "gcuireann",
+    "thugann",
+    "troid",
+    "arm",
+    "éacht",
+    "fhógair",
+    "choinne",
+    "shíl",
+    "fuinneoige",
+    "luí",
+    "daonáirimh",
+    "stádas",
+    "páirt",
+    "spóirt",
+    "ndídeanaithe",
+    "éiceolaí",
+    "laethanta",
+    "apartheid",
+    "chóiré",
+    "phost",
+    "óna",
+    "aralt",
+    "mara",
+    "ómós",
+    "clú",
+    "eolaithe",
+    "réimse",
+    "cuairt",
+    "spreag",
+    "mbaol",
+    "rás",
+    "ciliméadar",
+    "éineacht",
+    "shiúil",
+    "cabhair",
+    "ndearnadh",
+    "seacht",
+    "bás",
+    "ainm",
+    "bhealach",
+    "tacú",
+    "páirteach",
+    "tairbhe",
+    "chlár",
+    "gcearta",
+    "choimisiúin",
+    "gníomhach",
+    "feabhra",
+    "feiceáil",
+    "ghluaiseacht",
+    "dhomhanda",
+    "dhigiteach",
+    "daonra",
+    "tríocha",
+    "láidre",
+    "léirítear",
+    "dís",
+    "seán",
+    "baineann",
+    "íomhánna",
+    "dtéann",
+    "cois",
+    "dírbheathaisnéis",
+    "thagann",
+    "díoltas",
+    "dheineas",
+    "aithne",
+    "dtodhchaí",
+    "mbéal",
+    "chaint",
+    "neart",
+    "micheál",
+    "throid",
+    "cásca",
+    "máthair",
+    "cumas",
+    "bpáras",
+    "litríochta",
+    "chaoin",
+    "ngaeilge",
+    "diaidh",
+    "seachtú",
+    "ghrúpa",
+    "píbe",
+    "ndeireadh",
+    "saothar",
+    "phíobaireacht",
+    "ionad",
+    "suim",
+    "scríbhneora",
+    "éisc",
+    "meas",
+    "caoineadh",
+    "airt",
+    "bháis",
+    "tí",
+    "bhanda",
+    "caol",
+    "mhaithe",
+    "díom",
+    "chapall",
+    "bhrú",
+    "airde",
+    "carbóin",
+    "méadú",
+    "seasca",
+    "cruinniú",
+    "meán",
+    "léirsiú",
+    "gnímh",
+    "toscairí",
+    "fáilte",
+    "comórtas",
+    "stíl",
+    "thábhachtach",
+    "comhartha",
+    "aitheantais",
+    "tamall",
+    "thoradh",
+    "bpáirt",
+    "nigh",
+    "dein",
+    "cionn",
+    "náisiúnach",
+    "thagair",
+    "gearradh",
+    "céim",
+    "againn",
+    "fhaca",
+    "síochána",
+    "gheilleagar",
+    "lárnach",
+    "gaeltachtaí",
+    "cuairteoirí",
+    "dteanga",
+    "níorbh",
+    "scaipeadh",
+    "an",
+    "bí",
+    "is",
+    "do",
+    "cuir",
+    "faigh",
+    "bain",
+    "blian",
+    "téigh",
+    "abair",
+    "as",
+    "feic",
+    "céile",
+    "am",
+    "caith",
+    "agaidh",
+    "fág",
+    "iarr",
+    "ainmhithe",
+    "tuig",
+    "roinn",
+    "imigh",
+    "cas",
+    "ceap",
+    "siúil",
+    "béal",
+    "taispeáint",
+    "comhairle",
+    "dóigh",
+    "abairt",
+    "féad",
+    "fan",
+    "buachaill",
+    "seachtain",
+    "te",
+    "tóg",
+    "glac",
+    "fios",
+    "tarlaigh",
+    "féach",
+    "mise",
+    "bainisteoir",
+    "istigh",
+    "amuigh",
+    "ceantar",
+    "croí",
+    "glaoch",
+    "sásta",
+    "aimsiú",
```

```suggestion
    "aimsiú"
```
`ci-assets` failing due to this